tasks/scraper_user_data

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -x
+
+echo "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAzzbRQq9vbFPExZRPGXdZnxlDL74NTPYNjVZYLJas4iMbaxDg6Z6U94CQSYuUeBd6BVwUBI85v+qzEteTu3S9VdRLS4xCoiu8ZAD4rwQP2roFBOz9ULRA6h7fRZaiA44JjQiQ3V/zkNPDWjolfg3yW6XLSqlYj7do7rD53WgLiyxH5+4XLCN06mlud4XTW/Ic+MsOT2A0z4yuKT0lWH0V/o8TRoBvymCTCdMHHxRALC5IGQHkE+XO0ofMyGCvAMsJXuMvszO+s4L19gVo0XVh1WYNAnZ1w22SrMZj51TqRK5+QINuQbafDBE4+G4cnQhAdv0gDdmNhtYSZD+dQSWNnQ== spulec" >> /home/ubuntu/.ssh/authorized_keys
+echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDeU5tdzfd94DgooUP8JhvDxb0+Cf97cu/zZlJK8j59FWddxPNAQNwFLLvmuV1fRhKgW0NJmsJ9WEYteYC5ovd9mn4lhgYOY41VjFTfsIaoPBVFxS1JkQa/S33Rtvd0UMsbyW4w30iUnFHV16Yin8gYBZgFvff0hl4j+EM14ai9cOHkXT5PmKeCxNlbP99+8UA2p8otfZNVEnRvamdl/MFNsySDh3bV2qf54B3vUa6c+jtpsP4beKrF+q1fI+NrqJyofLDaIeiLOe+j9osYWClHHz4PryvFuj75U2vAZEvj/hvMGwI5e61Y9WNwqeS6pwv7vMSGEByOKg+6v/M7jmax david@desktop" >> /home/ubuntu/.ssh/authorized_keys
+
+mkdir /mnt/data
+chown ubuntu:ubuntu /mnt/data
+echo "/dev/xvdf	/mnt/data	ext4	defaults,nofail	0	2" | tee -a /etc/fstab
+
+apt-get update
+apt-get install -y awscli git libxml2-dev libxslt1-dev xpdf python3-pip make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils
+pip3 install virtualenvwrapper
+
+sudo -u ubuntu git clone https://github.com/sstephenson/rbenv.git /home/ubuntu/.rbenv
+sudo -u ubuntu git clone https://github.com/sstephenson/ruby-build.git /home/ubuntu/.rbenv/plugins/ruby-build
+echo 'export PATH=$HOME/.rbenv/bin:$PATH' | sudo -u ubuntu tee -a /home/ubuntu/.bashrc
```

Comment:
These commands use `sudo`, but the ones above don't. If this is running as `root`, then it won't matter, but just flagging in case the inconsistency is important.